src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,20 @@
+import Itemlist from "./ItemList";
+import "./todolist.scss";
+
+export type Todolist = {

```

Comment:
In here will be better to put a singular name, like `Item`, we're not typing a list but an item inside of it